$OpenBSD: patch-CMakeLists_txt,v 1.6 2020/07/31 23:09:46 bcallah Exp $

-DLTO=OFF in CONFIGURE_ARGS doesn't catch.
Don't do git here.

Index: CMakeLists.txt
--- CMakeLists.txt.orig
+++ CMakeLists.txt
@@ -25,7 +25,7 @@ if(BINARY_RELEASE)
   set(ASAN OFF)
   set(UBSAN OFF)
   set(DEBUG OFF)
-  set(LTO ON)
+  set(LTO OFF)
   set(DIST ON)
   set(FASTER OFF)
 endif()
@@ -40,14 +40,8 @@ if(NIGHTLY_BUILD)
   set(FASTER OFF)
 endif()
 
-include(CMake/git.cmake)
-get_git_tag(GIT_TAG)
-if(NOT CMAKE_BUILD_TYPE MATCHES "Release")
-  get_git_commit_hash(GIT_COMMIT_HASH)
-endif()
-
 project(DevilutionX
-  VERSION ${GIT_TAG}
+  VERSION 1.0.1
   LANGUAGES C CXX)
 
 if(LTO)
